src/algebraic_geometry/presheafed_space.lean

```diff
@@ -0,0 +1,173 @@
+-- Copyright (c) 2019 Scott Morrison. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Scott Morrison
+import category_theory.instances.Top.presheaf
+
+universes v u
+
+open category_theory
+open category_theory.instances
+open category_theory.instances.Top
```

Comment:
I'm thinking more and more that `Top` should just be in the `_root_` namespace. But that is not something for this PR. So ignore this comment.